Docker/AlmaLinux9/Dockerfile-externals

```
@@ -10,6 +10,9 @@ ARG SPACK_COMMIT=6cb16c39ab85fbc211e50be804fa7a15f24ccebc
 ARG COMMIT_SHA=e49a0639ff987a84fa0c6263e068776f9d7b6f67
 ARG GITHUB_REPOSITORY=madbaron/key4hep-spack
```

```suggestion
ARG GITHUB_REPOSITORY=key4hep/key4hep-spack
```
Looks like we have missed this at some point